templates/shared_services/gitea/template_schema.json

```diff
@@ -2,10 +2,32 @@
   "$schema": "http://json-schema.org/draft-07/schema",
   "$id": "https://github.com/microsoft/AzureTRE/templates/shared_services/gitea/template_schema.json",
   "type": "object",
-  "title": "Gitea Shared Service",
-  "description": "Provides Gitea shared service",
+  "title": "Gitea - Git Mirror",
+  "description": "The Gitea shared service is typically used for mirroring external Git repositories.",
   "required": [],
-  "properties": {},
+  "properties": {
```

Comment:
I don't understand why there weren't already here?

Reply:
And same for some of the other services.

Reply:
They exist as default properties in `api_app/schemas/workspace_service.json`` as every service needs them. We didn't forsee the need to set default values. We shoudl probably lint to ensure they exist for services as in time they need adding to them all.

Reply:
Issue https://github.com/microsoft/AzureTRE/issues/1695 covers